### ISSUE TITLE
Downloads master zip

### ISSUE THREAD

**Issue by sawdoctor** [DerGoogler/MMRL#37]
When I try and download any magisk module it downloads the master zip from GitHub which isn't flashable with Magisk unless I'm being extremely dumb - (extremely possible)

**Comment by DerGoogler** [comment]
Yeah. It's a known issue. This a API issue, to fix this we'll have to change our API which will happen in future.

**Comment by sawdoctor** [comment]
Thanks for taking the time to reply and thanks for your hard work. Really nice app with great UI and nice features. Look forward to making this my default app
